tests/validation/tests/v3_api/scripts/pipeline/Jenkinsfile_ontag_matrix

```diff
@@ -164,10 +164,18 @@ if ( true == via_webhook() && (!(rancher_version ==~ rancher_version_regex)) ) {
                       }
                       else if (rancher_version.startsWith("v2.6")){
                         if (!env.RKE_VERSION) {
-                          RKE_VERSION = "v1.3.13"
+                          RKE_VERSION = "v1.3.15"
                         }
                         if (!env.RANCHER_K3S_VERSION) {
-                          RANCHER_K3S_VERSION = "v1.24.2+k3s1"
+                          RANCHER_K3S_VERSION = "v1.24.6+k3s1"
+                        }
+                      }
+                      else if (rancher_version.startsWith("v2.7")){
+                        if (!env.RKE_VERSION) {
+                          RKE_VERSION = "v1.3.15"
```

Comment:
It is 1.4.0-rc. You can use 1.3.15 also. Can we change it to latest 1.4.0-rc instead? and change it to released version of 1.4.0 once 2.7.0 is released?

Reply:
As long as we don't forget I'm good with using the RC if that's preferred over the 1.3.15